pkg/reconciler/cronjobsource/cronjobsource.go

```diff
@@ -294,7 +299,7 @@ func (r *Reconciler) reconcileEventType(ctx context.Context, src *v1alpha1.CronJ
 
 	// Only create EventTypes for Broker sinks. But if there is an EventType and the src has a non-Broker sink
 	// (possibly because it was updated), then we need to delete it.
-	if ref := src.Spec.Sink.GetRef(); ref == nil || ref.Kind != "Broker" {
+	if sinkRef == nil || sinkRef.Kind != "Broker" {
```

Comment:
We won’t get this far if sinkRef is nil.

Reply:
ack, fixed.